test/command/suite/select/function/string/string_slice/nth/options/no_option_key.test

```diff
@@ -0,0 +1,10 @@
+plugin_register functions/string
+
+table_create Memos TABLE_HASH_KEY ShortText
+
+load --table Memos
+[
+{"_key": "Groonga"}
+]
+
+select Memos   --output_columns '_key, string_slice(_key, 2, 2, {})'
```

Comment:
How about `test/command/suite/select/function/string/string_slice/nth/options/empty.test`?

Reply:
I fixed it.